chapter_appendix-mathematics-for-deep-learning/distributions.md

```diff
@@ -87,7 +87,7 @@ We can sample an array of arbitrary shape from a Bernoulli random variable as fo
 
 ## Discrete Uniform
 
-The next commonly encountered random variable encountered is a discrete uniform.  For our discussion here, we will assume that it is supported on the integers $\{1, 2, \ldots, n\}$, however any other set of values can be freely chosen.  The meaning of the word *uniform* in this context is that every possible value is equally likely.  The probability for each value $i \in \{1, 2, 3, \ldots, n\}$ is $p_i = \frac{1}{n}$.  We will denote a random variable $X$ with this distribution as
+The next commonly random variable encountered is a discrete uniform.  For our discussion here, we will assume that it is supported on the integers $\{1, 2, \ldots, n\}$, however any other set of values can be freely chosen.  The meaning of the word *uniform* in this context is that every possible value is equally likely.  The probability for each value $i \in \{1, 2, 3, \ldots, n\}$ is $p_i = \frac{1}{n}$.  We will denote a random variable $X$ with this distribution as
```

Comment:
The phrase "the next commonly random variable encountered" is not grammatical.  I'd rephrase to "the next commonly encountered random variable is a discrete uniform.

Reply:
Ok!